src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
The name 'AreaCalculable' is a verb form which is not recommended for class/interface names. Consider using a noun or a noun phrase, like 'AreaCalculator', which clearly reflects the intent of the interface.